.github/workflows/git-tag.yml

```diff
@@ -0,0 +1,34 @@
+name: Git Tag
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'CHANGELOG.md'
```

Comment:
It seems changes to this file are pretty well-controlled. We can leverage this consistent commit pattern to only run this workflow when it's been updated, presumably by a changeset PR.